examples/utils/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "@solana/example-utils",
+    "version": "0.0.0",
+    "private": true,
```

Comment:
PSA: Let's NOT let this become a big kitchen sink. Examples should be as simple as possible and not obfuscate any setup.

Reply:
Agree. No abstraction of the actual steps. People should be able to understand the whole example by reading the one file.